### PR TITLE
add sample github actions integration to go, gradle and pytest

### PR DESCRIPTION
**Pull request by shibuiwilliam**
# What
- Example integration of Launchable to GitHub Actions in go-test, gradle and pytest.

### CODE REVIEW

File: docs/resources/integrations/go-test.md [hunk 1]
```diff
@@ -51,3 +51,65 @@ This creates a file called `launchable-subset.txt` that you can pass into your c
...
+          # Install launchable CLI.
+          python -m pip install --upgrade pip
+          pip install wheel setuptools_scm
+          pip install launchable
+
+          # Verify launchable command.
+          launchable verify
+
+          # Record build name.
+          launchable record build --name ${{ github.sha }} --source .
```
Comment (awilkes) [inline docs/resources/integrations/go-test.md:105]:
```suggestion
          launchable record build --name ${{ github.sha }} --source src=.
```

File: docs/resources/integrations/gradle.md [hunk 2]
```diff
@@ -120,3 +120,57 @@ Note: The **Gradle plugin for Android** requires a different command, because th
...
+          # Install launchable CLI.
+          python -m pip install --upgrade pip
+          pip install wheel setuptools_scm
+          pip install launchable
+
+          # Verify launchable command.
+          launchable verify
+
+          # Record build name.
+          launchable record build --name ${{ github.sha }} --source .
```
Comment (awilkes) [inline docs/resources/integrations/gradle.md:166]:
```suggestion
          launchable record build --name ${{ github.sha }} --source src=.
```

File: docs/resources/integrations/pytest.md [hunk 3]
```diff
@@ -242,3 +242,57 @@ This creates a file called `launchable-subset.txt` that you can pass into your c
...
+          # Install launchable CLI.
+          python -m pip install --upgrade pip
+          pip install wheel setuptools_scm
+          pip install launchable
+
+          # Verify launchable command.
+          launchable verify
+
+          # Record build name.
+          launchable record build --name ${{ github.sha }} --source .
```
Comment (awilkes) [inline docs/resources/integrations/pytest.md:288]:
```suggestion
          launchable record build --name ${{ github.sha }} --source src=.
```

**Review by awilkes — APPROVED** [review]
Looks good to me, but don't we also need to add the steps from https://docs.launchableinc.com/sending-data-to-launchable/ensuring-record-tests-always-runs ?

File: docs/resources/integrations/pytest.md (hunk 3 above)
Comment (shibuiwilliam) [reply docs/resources/integrations/pytest.md]: fixed.
https://github.com/launchableinc/cli/pull/483/commits/31647a28a7550b1769a65cd3f29d33f2548cc6d9

File: docs/resources/integrations/gradle.md (hunk 2 above)
Comment (shibuiwilliam) [reply docs/resources/integrations/gradle.md]: fixed.
https://github.com/launchableinc/cli/pull/483/commits/31647a28a7550b1769a65cd3f29d33f2548cc6d9

File: docs/resources/integrations/go-test.md (hunk 1 above)
Comment (shibuiwilliam) [reply docs/resources/integrations/go-test.md]: fixed.
https://github.com/launchableinc/cli/pull/483/commits/31647a28a7550b1769a65cd3f29d33f2548cc6d9

**Comment by shibuiwilliam** [comment]
@awilkes 
> Looks good to me, but don't we also need to add the steps from https://docs.launchableinc.com/sending-data-to-launchable/ensuring-record-tests-always-runs ?

`if: always` is needed when test step and record test step are separated. Since I wrote them in one step, we don't have to have `if: always` here.
I feel it is better to show variety of ways to install Launchable, so I made a separated version and `if always`.
https://github.com/launchableinc/cli/pull/483/commits/668126599728fea77b8c18efe18f81f8357ce3d2

**Pull request merged**